src/mscorlib/src/System/Reflection/RuntimeConstructorInfo.cs

```diff
@@ -462,18 +461,5 @@ public override Object Invoke(BindingFlags invokeAttr, Binder binder, Object[] p
             return RuntimeMethodHandle.InvokeMethod(null, null, sig, true);
         }
         #endregion
-
-        #region ISerializable Implementation
-        public void GetObjectData(SerializationInfo info, StreamingContext context)
-        {
-            throw new PlatformNotSupportedException();
```

Comment:
Got it. That sounds reasonable (somebody could have been casting to ISerializable on NetFX, but given that the interface isn't on the base types, that's past the promise of NS 2.0)